tests/conftest.py

```diff
@@ -44,6 +44,10 @@ def clean_notebook() -> nbformat.NotebookNode:
     """A clean notebook."""
     return read_notebook("clean.ipynb")
 
+@pytest.fixture()
+def clean_notebook_without_empty_cells() -> nbformat.NotebookNode:
+    """A clean notebook without empty cells."""
+    return read_notebook("clean_without_empty_cells.ipynb")
```

Comment:
CI is failing because this function should be surrounded by two blank lines. You can format the file with Black using `poetry run black tests/conftest.py`.

Reply:
Haven't install Poetry yet. I'll make sure to use that for this package.